mcs/build/tests.make

```diff
@@ -272,7 +272,7 @@ endif
 
 # This handles .excludes/.sources pairs, as well as resolving the
 # includes that occur in .sources files
-$(test_response_preprocessed): $(test_sourcefile) $(wildcard $(test_sourcefile_excludes)) $(GENSOURCES_CS)
+$(test_response_preprocessed): $(test_sourcefile) $(wildcard *_test.dll.sources) $(wildcard *_test.dll.exclude.sources) $(GENSOURCES_CS)
```

Comment:
it should be *test.dll.sources here as well so we catch xunit too

Reply:
@akoeplinger `$(test_response_preprocessed)` is only for nunit so we don't need xunit files here. Xunit already has this fix.

Reply:
ah right, I overlooked that part